draft-pauly-dprive-oblivious-doh.md

```diff
@@ -154,7 +154,13 @@ as an HTTP request using the POST method. Requests to the Proxy indicate
 which DoH server to use as a Target by specifying two variables: "targethost",
 which indicates the host name of the Target server, and "targetpath", which
 indicates the path on which the Target's DoH server is running. See
-{{request-example}} for an example request.
+{{request-example}} for an example request. Clients are configured with a Proxy
```

Comment:
I would define the URI template before its variables, and add some restrictions:

```
Oblivious DoH queries are created by the Client, and sent to the Proxy
as an HTTP request using the POST method. Clients are configured with a Proxy
URI Template {{!RFC6570}} which contains two variables: "targethost",
which indicates the host name of the Target server, and "targetpath", which
indicates the path on which the Target's DoH server is running. Examples  of
URI templates are shown below:

~~~
https://dnsproxy.example/dns-query{?targethost,targetpath}
https://dnsproxy.example/{targethost}/{targetpath}
~~~

The URI template MUST contain both the "targethost" and "targetpath" variables
exactly once, and  MUST NOT contain any other variables. The variables MUST be
within the path component of the URI.

See {{request-example}} for an example request.
```

Reply:
Good suggestion! Fixed.